apps/admin-gui/src/assets/i18n/en.json

```
@@ -2024,9 +2025,11 @@
       "EXPIRE_NEVER": "Never expire",
       "EXPIRATION_ERROR": "The date has to be in format yyyy-mm-dd",
       "SELECT_SPONSOR": "Choose sponsor for created user",
+      "SELECT_SPONSOR_COPY": "Choose target sponsor",
```

Just to stay consistent with other labels.
```suggestion
      "SELECT_SPONSOR_COPY": "Select target sponsor",
```